CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 5.0.2
+
+This release of Teleport contains a security fix.
```

Comment:
Should this be highlighted as a markdown warning? That's what we do for Gravity changelogs containing vuln. fixes, not sure if Teleport does this usually or not.

Reply:
None of the previous releases in the changelog use that style, so I'll keep it consistent.